r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
re: i18n stuff: You would want to translate formattable strings like so:

`_("%(number)s days until Thanksgiving") % dict(number=2)`. The formatting should be *outside* of the `_()` call for everything to work properly.